pkg/v1/tkg/client/common.go

```diff
@@ -144,6 +172,32 @@ func (c *TkgClient) validateDockerPrerequisites() error {
 	return nil
 }
 
+func (c *TkgClient) ValidateDockerResourcePrerequisites() error {
```

Comment:
Q: I'm not sure if we should add this here or in https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/tkg/client/validate.go . IMO it would be worth a quick discussion